python/requirements/ml/tune-test-requirements.txt

```diff
@@ -1,6 +1,6 @@
 -r dl-cpu-requirements.txt
 
-aim==3.17.5; python_version < "3.12"
+aim==3.23.0; python_version < "3.12"
```

Comment:
is this version change still needed for pyiceberg?

Reply:
Yup, there is a sqlalchemy dependency via this, so this is needed